specs/Compatibility/RuntimeCompatibilityOptions.md

```diff
@@ -0,0 +1,226 @@
+   released in a future Windows App SDK update or for you to implement an update in your app.
+
+Here's an example of using `RuntimeCompatibilityOptions` to specify a patch level and disable a
+specific change:
+
+```c#
+void ApplyRuntimeCompatibilityOptions() 
+{
+    var compatibilityOptions = new RuntimeCompatibilityOptions();
+    compatibilityOptions.PatchLevel1 = new WindowsAppRuntimeVersion(1,7,3);
+    compatibilityOptions.PatchLevel2 = new WindowsAppRuntimeVersion(1,8,2);
+    compatibilityOptions.DisabledChanges.Add(RuntimeCompatibilityChange.SampleApiCrashFix);
+    compatibilityOptions.Apply();
+}
+```
+
+RuntimeCompatibilityOptions must be applied early in the process before any other Windows App
+SDK APIs are called, or right after initializing the Windows App Runtime.
+
+`PatchLevel1` and `PatchLevel2` are simply two fields to set relevant patch levels. These needn't
+match any specific version of the Windows App Runtime nor be in a specific order, so it is valid
+to set `PatchLevel1` to 1.8.2 and `PatchLevel2` to 1.7.3, for example. And, in the above example,
+when updating the app to 1.9, you may choose to simply update `PatchLevel1` to 1.9.3 and leave
+`PatchLevel2` as 1.8.2.
+
+### Specifying RuntimeCompatibilityOptions in the app's project file
+
+You can also specify the patch level and disabled changes in the app's project file instead of
+directly calling the RuntimeCompatibilityOptions APIs. This approach has the advantage of ensuring the
+options are applied early at the proper timing. Here's an example of how to specify the patch
+level and disabled changes in the project file (such as `.csproj` or `.vcxproj`):
+
+```xml
+  <PropertyGroup>
+    <WindowsAppSDKRuntimePatchLevel>1.7.3</WindowsAppSDKRuntimePatchLevel>
+    <WindowsAppSDKRuntimePatchLevel2>1.8.2</WindowsAppSDKRuntimePatchLevel2>
+    <WindowsAppSDKDisabledChanges>SampleApiCrashFix, OtherSampleApiCrashFix</WindowsAppSDKDisabledChanges>
+  </PropertyGroup>
+```
+
+The `WindowsAppSDKDisabledChanges` property is a comma-separated list of `RuntimeCompatibilityChange`
+values to disable.
+
+### Behavior with no PatchLevel specified
+
+If no `PatchLevel1` or `PatchLevel2` is specified, or if neither value matches the major.minor
+version of the runtime being used, then the runtime will use the latest patch level. In other
+words, the runtime will run with all servicing changes enabled (just like how Windows App SDK
+worked before this API existed).
+
+## RuntimeCompatibilityOptions.PatchLevel1
+
+Optional patch level to use if the runtime version matches the major.minor version. If the runtime
+version does not match the specified major.minor version, this value is ignored.
+
+Instead of directly using this API, this patch level could be specified in the app's project file,
+like this:
+
+```xml
+  <PropertyGroup>
+    <WindowsAppSDKRuntimePatchLevel>1.7.3</WindowsAppSDKRuntimePatchLevel>
```

Comment:
What is our capitalization convention on `WindowsAppSDK`? By the standard capitalization rules, it would be `WindowsAppSdk`.

Reply:
In the WinAppSDK .props/.targets files, everything currently uses capital "DK" except for a single `<WindowsAppSdkFoundation>` property which is for internal-only testing (and should ideally be reworked to not be in the official WinAppSDK package).